index.html

```diff
@@ -51,21 +49,21 @@ <h2>Social Media Marketing</h2>
     <div class="benefits">
```

Comment:
This seems like a container on the side of the page and research in to what semantic HTML element can be used to replace the `div`